packages/flutter/example/pubspec.yaml

```diff
@@ -12,21 +14,19 @@ dependencies:
   cupertino_icons: ^1.0.2
   parse_server_sdk_flutter:
     path: ../
-  path_provider: ^2.0.1
-  sembast: ^3.0.0+4
-  shared_preferences: ^2.0.5
 
-dependency_overrides:
-  # Override to local mono-repo path so devs can test this repo
-  # against changes that they're making to other mono-repo packages.
-  parse_server_sdk:
-    path: ../../dart
+  path: ^1.8.3
```

Comment:
> ERR : Because every version of flutter_test from sdk depends on path 1.8.2 and flutter_plugin_example depends on path ^1.8.3, flutter_test from sdk is forbidden.


Can you change `path: ^1.8.3` to `path: ^1.8.2`

Reply:
Sure